atlp-rwanda/e-commerce-ones-and-zeroes-bn
# 187355104 two factor authentication

#### What does this PR do?

Two Factor Authentication for Sellers

#### Description of Task to be completed?

how the following end point working 

 . post  /api/user/registerUser
 .post   /api/user/loginUser
 .post   /api/user/2fa-verify

#### How should this be manually tested?

  . Registration: Sign up as a seller on the web application.
  . Login: Login with credentials and request 2FA.
  . Email: Retrieve the 2FA code from the email.
  . Authenticator: Use an authenticator app to get the TOTP token.
  . Postman: Verify the 2FA by making a POST request with user ID and token.

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/187355104

#### Screenshots (if appropriate)

![Screenshot 2024-05-22 174324](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/156195811/4c3dfa94-3ae6-4fa9-a335-cb6b27ce5ea8)
